modules/camera/base_camera.py

```diff
@@ -15,7 +15,7 @@ class BaseCameraDevice(abc.ABC):
     @classmethod
     @abc.abstractmethod
     def create(
-        cls, width: int, height: int
+        cls, width: int, height: int, config: object
```

Comment:
Type of config should be `camera_config.PiCameraConfig | camera_config.OpenCVCameraConfig`